feign/src/main/java/com/baeldung/core/ExampleApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.baeldung.core;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+
+@SpringBootApplication
```

Comment:
why we need this class? how was this functioning in the spring-cloud-modules module ?

Reply:
Because the code migrated from that module runs with this class.